src/coord/src/coord.rs

```diff
@@ -550,12 +590,8 @@ impl Coordinator {
     // backward). This downgrades the capabilities of all tables, which means that
     // all tables can no longer produce new data before this timestamp.
     fn advance_local_inputs(&mut self) {
```

Comment:
Should this be `advance_tables` to line up with other uses of tables above? Ideally, we get this to `advance_tables(&mut self, timeline: TimelineId)` but I think the dataflow layer doesn't have the commands for that either yet.

Reply:
Done. Took the opportunity to fix the comment as well, since this function no longer consults the wall clock.